src/app/modules/myinfo/myinfo.constants.ts

```diff
@@ -31,6 +44,16 @@ export const MYINFO_COOKIE_OPTIONS = {
   secure: !config.isDev,
 }
 
+/**
+ * Settings for the MyInfo auth code cookie
+ */
+export const MYINFO_AUTH_CODE_COOKIE_OPTIONS = {
+  // Important for security - access token cannot be read by client-side JS
```

Comment:
nit: should be auth code, not access token

Reply:
oops tq

Reply:
357adda2